components/TextFields/src/MDCTextInputControllerFullWidth.m

```diff
@@ -424,6 +424,22 @@ - (void)setCharacterCountViewMode:(UITextFieldViewMode)characterCountViewMode {
   }
 }
 
+- (void)setDisabledColor:(UIColor *)disabledColor {
```

Comment:
I know that __unused can be used to suppress compiler warnings (not sure why it wouldn't be in this case), but I think it's also a healthy call-out to creators and maintainers that there is an argument being discarded.  Unused parameters are often completely innocuous, I just think it's a nice-to-have for source code bookkeeping.